aztarna/ros/sros/scanner.py

```diff
@@ -40,7 +39,7 @@ def __init__(self):
             sros_host = None
             logger.warning('Connecting to {}:{}'.format(address, master_port))
             try:
-                master_address, port, master_cert = await get_sros_certificate(address, master_port, timeout)
+                port, master_cert = await get_sros_certificate(address, master_port, timeout)
```

Comment:
I think this is wrong.

Reply:
The warning was, unused variable. But, if should be like this I'm going to revert the change.